src/pages/my-cooperations/MyCooperations.tsx

```diff
@@ -133,20 +126,20 @@ const MyCooperations = () => {
         view={itemsView}
         withoutSort={showTable}
       />
-      {loading ? (
+      {isLoading ? (
         <Loader pageLoad size={50} />
       ) : (
         <>
           <CooperationContainer
-            getCooperations={getCooperations}
-            items={response.items}
+            getCooperations={refetch}
+            items={data?.items ?? []}
             showTable={showTable}
             sort={sortOptions}
           />
           <AppPagination
             onChange={handleChangePage}
             page={page}
-            pageCount={Math.ceil(response.count / itemsPerPage)}
+            pageCount={data?.count ? Math.ceil(data.count / itemsPerPage) : 0}
```

Comment:
Replace the 0 with a 1 as the page can't start with a 0 but I'm not sure on this one 😓